normandy/recipes/checks.py

```diff
@@ -50,8 +51,11 @@ def recipe_signatures_are_correct(app_configs, **kwargs):
         signed_recipes = list(Recipe.objects.exclude(signature=None).select_related("signature"))
     except (ProgrammingError, OperationalError, ImproperlyConfigured) as e:
         errors.append(Info(f"Could not retrieve recipes: {e}", id=INFO_COULD_NOT_RETRIEVE_RECIPES))
-    else:
+        return errors
+
+    try:
         for recipe in signed_recipes:
+            print("@@@", recipe.canonical_json)
```

Comment:
did you intend to have this in here?

Reply:
I'll wait for @mythmon to respond to this before we merge.

Reply:
Oops. Fixed.